environment_py2.yml

```diff
@@ -0,0 +1,45 @@
+# environment_py2.yml
```

Comment:
the `environment_py2.yml` file is meant to replace the `tethys_conda_env.yml` file. Once we add Python 3 support for then we can create a `environment_py3.py` and make any necessary changes (which may only be changing the python version from 2 to 3). If we are dropping Python 2 support once we move to 3 then maybe we should just call this `environment.yml` and change it once we move to 3.